aws-ts-eks-migrate-nodegroups/iam.ts

```diff
@@ -0,0 +1,51 @@
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+import * as iam from "./iam";
+
+const managedPolicyArns: string[] = [
+    "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy",
```

Comment:
question on these - if these are know policies that we need to tell people to use, should we add these to `aws.iam.ManagedPolicies` so that users get auto-complete on the correct policies to use?

Reply:
We've previously discussed this topic and the short of it is that this approach ultimately allows users full control over the roles, and maps well to our desired state model. See these threads for more info:

- https://github.com/pulumi/pulumi-eks/pull/98#issuecomment-479978253
- https://github.com/pulumi/pulumi-eks/pull/98#issuecomment-480452516